<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>711</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>80</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>109</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="variableView">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QRadioButton" name="viewTemperatureRadio">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="viewPressureRadio">
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="viewVelocityRadio">
       <property name="text">
        <string>Velocity</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>270</y>
      <width>111</width>
      <height>221</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="interactionType">
     <item>
      <widget class="QRadioButton" name="wallSourceRadio">
       <property name="text">
        <string>Wall</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="pressureSourceRadio">
       <property name="text">
        <string>Pressure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="temperatureSourceRadio">
       <property name="text">
        <string>Temperature</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="GraphicsView" name="fluidWindow" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>641</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
